app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,78 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * 상품 관련 CRUD 로직을 담당
+ */
+@Service
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    /**
+     * 상품 목록 조회
+     * @return 조회한 상품 목록
+     */
```

Comment:
메소드에 붙이는 JavaDoc의 경우 **리턴값**에 대한 설명이 가장 중요합니다.
```suggestion
    /**
     * 상품 목록을 조회해 리턴합니다.
     * @return 조회한 상품 목록
     */
```

다음은 `java.io.File` 클래스의 일부를 스크린샷으로 찍은 거에요. `Returns`로 문장이 시작하고 있죠? `~~을 리턴합니다.`라는 표현이 엄청나게 중요한 이유는 별 거 없습니다. 내가 처음 보는 메소드가 "어떤 입력을 받아" "어떤 결과를 내놓는지" 이해시킬 수 있다면 가장 빨리 응용하는 것도 가능할 겁니다.

![image](https://user-images.githubusercontent.com/1855714/119844566-8335fe80-bf43-11eb-9ae1-16b4563722fc.png)

Reply:
결국 JavaDoc에서 첫 설명이 가장 중요하네요. 좀 더 고민해보며 작성 해보겠습니다!